frontend/lib/identification/verification_workflow/dialogs/positive_verification_result_dialog.dart

```diff
@@ -28,19 +53,43 @@ class PositiveVerificationResultDialog extends StatelessWidget {
       builder: (result, {refetch, fetchMore}) {
         final data = result.data;
         final region = result.isConcrete && data != null ? regionsQuery.parse(data).regionsByIdInProject[0] : null;
+        final bool isUncheckedStaticQrCode = !isChecked && widget.hasStaticVerificationCode;
         return InfoDialog(
-          title: localization.positiveVerificationDialogTitle,
-          icon: Icons.verified_user,
-          iconColor: Colors.green,
-          child: IdCard(
-            cardInfo: cardInfo,
-            region: region != null ? Region(region.prefix, region.name) : null,
+          title: isUncheckedStaticQrCode ? "Prüfung notwendig" : localization.positiveVerificationDialogTitle,
```

Comment:
```suggestion
          title: isUncheckedStaticQrCode ? "Prüfung nötig" : localization.positiveVerificationDialogTitle,
```
(Increasing the chance that it fits in one line)